backend/src/main/java/com/woowacourse/moamoa/study/service/StudyService.java

```diff
@@ -74,4 +77,22 @@ public void autoUpdateStatus() {
             study.changeStatus(now);
         }
     }
+
+    public void updateStudy(Long githubId, Long studyId, StudyRequest request) {
+        Study study = studyRepository.findById(studyId)
+                .orElseThrow(StudyNotFoundException::new);
+        Member member = memberRepository.findByGithubId(githubId)
+                .orElseThrow(MemberNotFoundException::new);
+
+        checkOwner(study, member);
+
+        study.update(request.mapToContent(), request.mapToRecruitPlan(), request.mapToAttachedTags(),
+                request.mapToStudyPlanner(LocalDate.now()));
+    }
+
+    private void checkOwner(Study study, Member member) {
+        if (!Objects.equals(study.getParticipants().getOwnerId(), member.getId())) {
+            throw new UnauthorizedException("스터디 방장만이 스터디를 수정할 수 있습니다.");
+        }
+    }
```

Comment:
```suggestion
    private void checkOwner(Study study, Member member) {
        if (!study.isOwner(member.getId())) {
            throw new UnauthorizedException("스터디 방장만이 스터디를 수정할 수 있습니다.");
        }
    }
```
와 같이 메시지를 던지도록 수정해보는건 어떨까요?

Reply:
피드백 반영해 보았습니다!